lib/features/composer/presentation/widgets/mobile/tablet_bottom_bar_composer_widget.dart

```diff
@@ -71,6 +75,18 @@ class TabletBottomBarComposerWidget extends StatelessWidget {
               : AppLocalizations.of(context).turnOnRequestReadReceipt,
             onTapActionCallback: requestReadReceiptAction,
           ),
+          if (onOpenAIScribe != null) ...[
+            const SizedBox(width: TabletBottomBarComposerWidgetStyle.space),
+            TMailButtonWidget.fromIcon(
+              key: aiScribeButtonKey,
+              icon: imagePaths.icSparkle,
+              borderRadius: TabletBottomBarComposerWidgetStyle.iconRadius,
+              padding: TabletBottomBarComposerWidgetStyle.iconPadding,
+              iconSize: TabletBottomBarComposerWidgetStyle.iconSize,
+              tooltipMessage: AppLocalizations.of(context).aiAssistant,
+              onTapActionCallback: onOpenAIScribe!,
+            ),
```

Comment:
We should create a separate widget for it so we can reuse it in multiple places.